hydrocron/db/io/swot_reach_node_shp.py

```diff
@@ -191,7 +193,10 @@ def parse_from_filename(filename):
             filename_components[9],
             '%Y%m%dT%H%M%S').strftime('%Y-%m-%dT%H:%M:%SZ'),
         'crid': filename_components[10],
-        'collection_shortname': collection
+        'collection_shortname': collection,
+        'collection_version': collection_version,
+        'granuleUR': filename,
```

Comment:
Is `granuleUR` == `filename`? I thought generally granuleUR was the filename without the extension but I'm not certain that is a guarantee either. Perhaps we should just call this column `filename` to not risk confusing the two concepts?

Reply:
Usually granuleUR doesn't include the extension, but for this specific collection it happens to match the full filename, I'm not sure why. OPS does their monitoring based on comparisons to granuleUR in CMR, which is why I chose to name it granuleUR here so that it's a direct comparison. This field isn't exposed in the API response, it's currently just for db reconciliation so I'm inclined to keep using the language the OPS team requested, but can change to filename if there's a strong preference

Reply:
Ok that makes sense then. My concern was mostly avoiding misrepresenting the data in case granuleUR != filename. If that convention ever does change for this specific collection, we're bound to get questions about that. But since that is the current status quo no complaints from me.